CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.110.1] - 2022-07-25
+
```

Comment:
Do we want to also add new nodes and stuff (ie., [BETWEEN operator](https://github.com/transform-data/metricflow/pull/168)) or are these specifically for client facing features

Reply:
Doh, sorry. I missed that one.